packages/theming/theme-tokens/src/getShadowTokens.android.ts

```diff
@@ -0,0 +1,14 @@
+
+import lightShadowTokens from '@fluentui-react-native/design-tokens-android/light/tokens-shadow.json';
+import darkShadowTokens from '@fluentui-react-native/design-tokens-android/dark/tokens-shadow.json';
+import { AppearanceOptions } from '@fluentui-react-native/theme-types';
+
+export function getShadowTokens(mode: AppearanceOptions) {
+  if (mode === 'light') {
+    return lightShadowTokens;
+  } else if (mode === 'dark') {
+    return darkShadowTokens;
+  }
+
+return lightShadowTokens;
```

Comment:
Could maybe add an assert that we don't reach this line? Are there other appearance modes on Android

Reply:
There are no other appearance options in Android - only light and dark.